packages/google_maps_flutter/google_maps_flutter/example/ios/Runner/Runner-Bridging-Header.h

```diff
@@ -0,0 +1 @@
+#import "GeneratedPluginRegistrant.h"
```

Comment:
remove

Reply:
I will revert the `example`. I think `flutter create` was run inside the `example` dir, and it added a lot of stuff that is not needed for this PR.